submission.json

```diff
@@ -21,10 +21,8 @@
         "id": 3361,
         "name": "Accepted",
         "max_grade": 2,
-        "pass_grade": 2,
         "grade_labels": [
-          { "grade": 1, "label": "No" },
-          { "grade": 2, "label": "Yes" }
+          { "grade": 1, "label": "Yes" }
```

Comment:
The hash is missing a grade label for `grade_2`. As we have given `2` as the `max_grade` I think its good we add grade label for `2`.

Reply:
Missed that. Changed `max_grade` to 1. As that would be the default once the failure grades get removed.